crux-mir/README.md

```diff
@@ -56,12 +56,47 @@ versions corresponding to the last two `crux-mir` releases.
 [mir-json-readme]: https://github.com/GaloisInc/mir-json#readme
 
 
-## Setup: Build and Install
+## Setup
```

Comment:
Why remove the "Build and Install" part of this section header? I think it's important to include to distinguish it from the "Setup: `mir-json`" section that precedes it.

Reply:
I reworded the section, please have a look